tests/containers/dataDictionary/dataDictionaryContainer-test.jsx

```diff
@@ -5,39 +5,37 @@
 
 import React from 'react';
 import { mount } from 'enzyme';
-import DataDictionaryContainer from 'containers/bulkDownload/dictionary/DataDictionaryContainer';
-import { mockDictionary } from '../mockData';
+import DataDictionaryContainer from 'containers/dataDictionary/dataDictionaryContainer';
 
-// mock the bulkDownload helper
-jest.mock('helpers/bulkDownloadHelper', () => require('../mockBulkDownloadHelper'));
+import { mockApiCall } from '../../testResources/mockApiHelper';
+import * as api from '../../../src/js/helpers/apiRequest';
```

Comment:
these imports should be absolute. 

We have rules in our webpack config to make `helpers` resolve to `~/src/js/`

Reply:
thanks, done